src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubConfiguration.cs

```diff
@@ -230,12 +231,16 @@ EventProcessorHost IEventHubProvider.GetEventProcessorHost(string eventHubName,
                     sb.EntityPath = null; // need to remove to use with EventProcessorHost
                 }
 
-                EventProcessorHost host = new EventProcessorHost(
-                   eventProcessorHostName,
-                   actualPath,
-                   consumerGroup,
-                   sb.ToString(),
-                   storageConnectionString);
+                string @namespace = GetServiceBusNamespace(creds.EventHubConnectionString);
+                string containerName = EventHubLeaseFactory.GetContainerName(eventHubName, @namespace, storageConnectionString);
```

Comment:
We need tests for this code - tests that exercise this blob code and verify that the correct blob is created, etc. Also a test that verifies concurrency is handled correctly - e.g. three Tasks that run concurrently, and verify that each gets the same container name.